api-samples/history/showHistory/typedUrls.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Recently Typed URLs</title>
+    <style>
+      body {
+        min-width: 250px;
+      }
+    </style>
+    <script src="typedUrls.js"></script>
```

Comment:
Can you rename the script file to use lowercase and dashes? Same with the HTML file. Thanks!

Also, can you move the script tag into the body?

[File name style guide
](https://developers.google.com/style/filenames#:~:text=Guidelines%20for%20names,-Make%20file%20and&text=Use%20hyphens%2C%20not%20underscores%2C%20to,page%20names%20such%20as%20document1.
)